apperals-frontend/src/components/navigation/ButtonRow.css

```diff
@@ -3,61 +3,91 @@
     width: 100%;
     max-width: 1250px;
     margin: 0 auto;
-    padding: 10px;
+    padding: 20px;
     display: flex;
     gap: 20px;
     flex-direction: column;
     align-items: center;
     position: relative;
 }
 
+/* button bar */
 .button-row {
     display: flex;
-    /* justify-content: space-evenly; */
+    justify-content: center; 
     gap: 20px;
+    flex-wrap: wrap;  /* Allow buttons to wrap on smaller screens */
 }
 
 button {
     padding: 10px 20px;
     font-size: 1rem;
     cursor: pointer;
-    width: 160px;
-    height: 35px;
+    width: 170px;
+    height: 50px;
     border: none;
     outline: none;
     background-color: #f6f6f6;
 }
 
+/* title bar */
 .title-row {
     display: flex;
     justify-content: space-between;
+    flex-direction: row; /* Stack title and button row on top of each other */
     width: 100%;
     align-items: center;
     margin-bottom: 20px;
+    gap: 20px;
 }
 
 .title {
     font-size: 32px;
     font-weight: bold;
-    white-space: nowrap;
+    text-align: center;
+    margin-bottom: 20px;
+    /* white-space: nowrap; */
 }
 
+/* Hamburger menu */
+.hamburger-icon {
+    display: none;
+    font-size: 30px;
+    cursor: pointer;
+    padding: 10px;
+    background-color: transparent;
+    border: 2px solid red;
+    position: absolute; /* Position it in the top-left corner */
+    top: 20px; 
+    left: 20px;
+  }
+
+/* Mobile menu hidden when not mobile screen */
+.mobile-menu {
+    display: none;
+    flex-direction: column;
+    align-items: center;
+    gap: 20px;
+    margin-top: 20px;
+  }
+
+/* background image static */
```

Comment:
All the classes related to image or the Banner Segment would go into another component